sanic/server/websockets/frame.py

```diff
@@ -96,6 +96,7 @@ async def get(self, timeout: Optional[float] = None) -> Optional[Data]:
         If ``timeout`` is set and elapses before a complete message is
         received, :meth:`get` returns ``None``.
         """
+        completed: bool
```

Comment:
Is this related to this PR at all?

Reply:
It was a typing error that was caught when I undid the upper limit on mypy.

Reply:
I see